### PR TITLE
add tracingTag filter

### PR DESCRIPTION
**Pull request by NorthFury**
This adds a simple filter for adding a custom opentracing tag.

### CODE REVIEW

**Comment by NorthFury** [comment]
:+1:

File: filters/tracing/tag_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	f.Request(ctx)
+
+	v, ok := span.Tags[tagName]
+	if !ok {
+		t.Error("tag was not set")
```
Comment (szuecs) [inline filters/tracing/tag_test.go]: please add `tagName` to the error

File: filters/tracing/tag_test.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	}
+
+	vs, ok := v.(string)
+	if !ok || vs != tagValue {
+		t.Error("invalid header value was copied")
```
Comment (szuecs) [inline filters/tracing/tag_test.go]: Error should contain the value of `vs` and `tagValue`.
`t.Errorf("invalid header value '%s' != '%s' was copied", vs, tagValue)`

**Comment by szuecs** [comment]
:+1:

**Pull request merged**